Demo/src/main/java/com/paypal/android/utils/BundleParcelableExt.kt

```diff
@@ -0,0 +1,12 @@
+package com.paypal.android.utils
+
+import android.os.Build
+import android.os.Build.VERSION.SDK_INT
+import android.os.Bundle
+import android.os.Parcelable
+
+// Ref: https://stackoverflow.com/a/73311814
+inline fun <reified T : Parcelable> Bundle.parcelable(key: String): T? = when {
```

Comment:
Any idea why the OG of this was deprecated? Just curious more than anything else.

Reply:
Yeah i'm not sure honestly. My guess would be improved type safety since the class needs to be declared explicitly rather than having to cast the value at runtime.

Reply:
Gotcha - that makes sense